.github/workflows/npm-publish-all-packages-canary.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '16.x'
+          node-version: '18.x'
```

Comment:
Using just `18` produces the same effect ;)

Reply:
fair, tbh just used the same syntax as other places!